libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/offroute/OffRouteDetectorTest.java

```diff
@@ -89,6 +89,28 @@ public void isUserOffRoute_AssertTrueWhenTooFarFromStep() throws Exception {
     assertTrue(isUserOffRoute);
   }
 
+  @Test
+  public void isUserOffRoute_StepPointSize() throws Exception {
+    RouteProgress routeProgress = buildDefaultTestRouteProgress();
+    Point stepManeuverPoint = routeProgress.directionsRoute().legs().get(0).steps().get(0).maneuver().location();
+    removeAllButOneStepPoints(routeProgress);
+
+    Location firstUpdate = buildDefaultLocationUpdate(-77.0339782574523, 38.89993519985637);
+    offRouteDetector.isUserOffRoute(firstUpdate, routeProgress, options);
+
+    Point offRoutePoint = buildPointAwayFromPoint(stepManeuverPoint, 50, 90);
+    Location secondUpdate = buildDefaultLocationUpdate(offRoutePoint.longitude(), offRoutePoint.latitude());
+
+    boolean isUserOffRoute = offRouteDetector.isUserOffRoute(secondUpdate, routeProgress, options);
+    assertFalse(isUserOffRoute);
+  }
+
+  private void removeAllButOneStepPoints(RouteProgress routeProgress) {
```

Comment:
NIT - Could you place this method at the bottom with the rest of the `private` methods?

Reply:
Sure :).

BTW: I checked before first committing this method, there are no other private methods at the bottom ;).

Reply:
I pointed it out because we normally organize the code in this particular order :point_right: `public`, `protected`, `private`, etc. trying to locate them in order of calling.
Especially here (in tests) so we don't have `private` methods mixed with the tests 😉

Reply:
Thanks for the clarification :).